models/cable/README.md

```diff
@@ -0,0 +1,64 @@
+A generic template for adding a new model to PEcAn
```

Comment:
The entirety of this text should be deleted and replaced with information about CABLE (documentation url, citation,  how to get and compile the CABLE code, etc) and the coupler (summary of any key or unusual design decisions that an end user or future developer should know about)